mnemosyne/core/config.py

```
@@ -22,7 +22,7 @@
 import os
 import threading
 from pathlib import Path
-from typing import Any, Dict, List, Optional, Set, Tuple
+from typing import Any, Dict, List, Optional, Set
```

Ruff (0.15.21)

[warning] `typing.Dict` is deprecated, use `dict` instead

(UP035)